### DISCUSSION TITLE
Why do I fail to enter the default user name and password for the first login?

### DISCUSSION THREAD

**Discussion by changchenwang** [apache/dolphinscheduler#15453] [General]
Hello everyone! Could you please tell me why do I fail to enter the default user name and password for the first login? (admin xx)
Thanks!

![image](https://github.com/apache/dolphinscheduler/assets/64242923/c8234cb3-cea9-45d1-8453-28c37e35b3d7)

**Comment by dzygoon** [comment]
reason has two:
1. check your database user table.
2. check your input password.

or in browser F12 check error msg.
